deploy/topic-klage-vedtak-fattet-dev.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  name: vedtak-fattet
+  namespace: klage
+  labels:
+    team: klage
+  annotations:
+    dcat.data.nav.no/title: "Vedtak fattet på klage"
+    dcat.data.nav.no/description: "Fullførte vedtak på klageinstans som er klare for førsteinstans postes på dette topicet."
+spec:
+  pool: nav-dev
+  config:  # optional; all fields are optional too; defaults shown
+    cleanupPolicy: delete  # delete, compact
+    minimumInSyncReplicas: 1
+    partitions: 1
+    replication: 3  # see min/max requirements
+    retentionBytes: -1  # -1 means unlimited
+    retentionHours: 72  # -1 means unlimited
```

Comment:
72 timer er 3 døgn. Det er ganske lite, klientene risikerer å miste et svar hvis de er nede over en langhelg eller noe sånt. Vi legger opp til å returnere ganske små meldinger, det kan neppe gjøre noe om vi skrur opp til en to-tre uker eller måneder for den saks skyld?

Reply:
Jepp. Skal ta og tune litt på denne og fikse noen andre småting.